### DISCUSSION TITLE
Data importer install without shell access

### DISCUSSION THREAD

**Discussion by kriswdue** [firefly-iii/firefly-iii#7086] [Firefly III] [answered]
Hi!

I'd like to install data-importer onto a web host where I have no shell access. Is there a way to do this?

- kriswdue

**Comment by JC5** [comment, marked as answer]
Perhaps you can install it somewhere offline and then upload the files?

**Reply by kriswdue** [reply]
Thank you for the quick response -  I'll try that!  :)

- Kriswdue

**Reply by kriswdue** [reply]
This worked fine! Did a local virtualized installation and just copied all of it to the server. Took me a while to figure out that I had to point the DocumentRoot to ./public, but otherwise it seems to work fine. 

Thank you for your fantastic work on Firefly and your follow-up here!!
